js/pages/myToDoList.js

```diff
@@ -336,7 +336,7 @@ const renderMainBody = (data, collections, tab) => {
     }
 
     if(modules['Covid-19'].enabled) {
-        toDisplaySystem.unshift({'body':['Covid-19']});
+        toDisplaySystem[1].body.unshift('Covid-19');
```

Comment:
Is it possible that `toDisplaySystem[1]` doesn't point to the body of SSN?  Based on the logic, there're several scenarios this could happen. For example,  if "Biospecimen Survey" and "Menstrual Cycle" are enabled, their data will be added to the start of array, and `toDisplaySystem[1]` doesn't refer to SSN. Please disregard if these scenarios won't happen.

Reply:
@we-ai already updated. Please help recheck again.